MetaMask/metamask-extension
# feat: add Confirmation Signature Redesign ui_customization event prop

## **Description**

Adds ui_customization: 'redesigned_confirmation' to the Signature <Approved|Rejected|Canceled> events

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/24182?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/23555

## **Manual testing steps**

1. set `ENABLE_CONFIRMATION_REDESIGN=true` in .metamaskrc
2. enable Experimental > Redesign Confirmation in settings
3. enable Participate in MetaMetrics in settings
4. test signatures (excluding Sign-in with Ethereum)
5. observe Signature <Rejected|Canceled|Approved> event with new ui_customization


## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
